awx/api/permissions.py

```diff
@@ -227,3 +227,12 @@ class IsSuperUser(permissions.BasePermission):
 
     def has_permission(self, request, view):
         return request.user and request.user.is_superuser
+
+
+class InstanceGroupTowerPermission(ModelAccessPermission):
```

Comment:
Leveraged DRF permissions because I wanted the `OPTIONS` action list to be accurate.